index.html

```diff
@@ -12,32 +12,20 @@
 <body>
   <div class="container">
     <div class="row">
-      <div class="col-lg-6 col-md-12">
-        <!-- ko foreach: toDoLists -->
-        <div class="row toDoList">
-          <!-- ko component: 'toDoList' -->
-          <!-- /ko -->
-        </div>
-        <button class="btn btn-block btn-danger" data-bind="click: $parent.removeToDoList">Remove</button>
+      <!-- ko foreach: widgets -->

```

Comment:
We have a single bootstrap row. We loop through the widgets in the widgets array.